slapp/utils/merge_utils.py

```diff
@@ -0,0 +1,186 @@
+from typing import List, Union, Optional, TypedDict
+from pathlib import Path
+import copy
+import jsonlines
+import boto3
+from urllib.parse import urlparse
+import tempfile
+import shutil
+import logging
+import json
+
+from slapp.transfers.utils import read_jsonlines
+
+
+class MergingException(Exception):
+    pass
+
+
+class WorkerAnnotation(TypedDict):
+    workerId: str
+    roiLabel: str
+
+
+class RecordProject(TypedDict):
+    sourceData: str
+    majorityLabel: str
+    workerAnnotations: List[WorkerAnnotation]
+
+
+def compute_majority(labels):
+    """return majority labels or None if tied
```

Comment:
Could you add more information (and typing) about how `labels` is organized?

Reply:
I'll add docstrings... sorry about that

Reply:
done